src/complete.jl

```diff
@@ -182,7 +188,7 @@ function complete_polynomial_impl!{T,N,D}(out::Type{Vector{T}}, z::Type{Vector{T
                             notD_d = notD_{d+1}*z[i_d]
                         end
 
-                        out[ix] = coeff_d * z[D]^(coeff_d-1) * notD_d
+                        out[ix] = coeff_d == 0 ? zero(T) : coeff_d * z[D]^(coeff_d-1) * notD_d
```

Comment:
Ah -- I forgot I made this change too when I submitted it. Thanks for checking it.

If `coeff_d` is 0 then `coeff_d` < 0 and if `z[D]` is `zero(T)` then it can't be raised to a negative power. This change makes sure that the code won't break when an element of `z` is 0.